source/_integrations/generic.markdown

```diff
@@ -15,7 +15,12 @@ Home Assistant will serve the images via its server, making it possible to view
 
 ## Configuration
 
-To enable this camera in your installation, add the following to your `configuration.yaml` file:
+Home Assistant offers the Generic IP Camera through **Configuration -> Integrations -> Generic IP Camera**.
+
+You must enter a URL in at least one of the fields **Still Image URL** or **Stream Source URL**, the others are optional. Click next to continue.
+
+Alternatively, Home Assistant can load this integration via `configuration.yaml`; add the following to your `configuration.yaml` file:
+
 
```

Comment:
We can drop the whole yaml section now

Reply:
Done.  I kept the examples because I thought they were really good, but removed the yaml parts of them (so they now just describe what to type into the UI).